tests/config.py

```diff
@@ -44,6 +44,7 @@ class Simple(pexConfig.Config):
     d = pexConfig.DictField("dict test", str, str, default={"key":"value"}, 
             itemCheck=lambda x: x.startswith('v'))
     n = pexConfig.Field("nan test", float, default=float("NAN"))
+    m = pexConfig.Field(None, int, optional=True)
```

Comment:
The HSC commit you're cherry-picking this from is described as "allow None as documentation string". Is that actually something we want? It doesn't sound like a very good idea.